manual/installation.tex

```diff
@@ -196,71 +203,89 @@ \subsection{Configuration options}
-                   CMAKE_CXX_FLAGS_RELEASE, and CMAKE_CXX_FLAGS_RELWITHDEBINFO
-\end{verbatim}
+\verbatimfont{\footnotesize}%
+\begin{verbatim}
+QMC_CUDA              Enable CUDA and GPU acceleration (1:yes, 0:no)
+QMC_COMPLEX           Build the complex (general twist/k-point) version (1:yes, 0:no)
+QMC_MIXED_PRECISION   Build the mixed precision (mixing double/float) version
+                      (1:yes (GPU default), 0:no (CPU default)).
+                      The CPU support is experimental.
+                      Use float and double for base and full precision.
+                      The GPU support is quite mature.
+                      Use always double for host side base and full precision
+                      and use float and double for CUDA base and full precision.
+ENABLE_TIMERS         Enable fine-grained timers (1:yes, 0:no (default)).
+                      Timers are off by default to avoid potential slowdown in small
+                      systems. For large systems (100+ electrons) there is no risk.
+ENABLE_SOA            (Experimental) Enable CPU optimization and GPU support using
+                      Structure-of-Array (SoA) datatypes (1:yes, 0:no (default)).
+\end{verbatim}
+\normalsize
+\item General build options
+\verbatimfont{\footnotesize}%
+\begin{verbatim}
+CMAKE_BUILD_TYPE     A variable which controls the type of build
+                     (defaults to Release). Possible values are:
+                     None (Do not set debug/optmize flags, use
+                     CMAKE_C_FLAGS or CMAKE_CXX_FLAGS)
+                     Debug (create a debug build)
+                     Release (create a release/optimized build)
+                     RelWithDebInfo (create a release/optimized build with debug info)
+                     MinSizeRel (create an executable optimized for size)
+CMAKE_C_COMPILER     Set the C compiler
+CMAKE_CXX_COMPILER   Set the C++ compiler
+CMAKE_C_FLAGS        Set the C flags.  Note: to prevent default
+                     debug/release flags from being used, set the CMAKE_BUILD_TYPE=None
+                     Also supported: CMAKE_C_FLAGS_DEBUG,
+                     CMAKE_C_FLAGS_RELEASE, and CMAKE_C_FLAGS_RELWITHDEBINFO
+CMAKE_CXX_FLAGS      Set the C++ flags.  Note: to prevent default
+                     debug/release flags from being used, set the CMAKE_BUILD_TYPE=None
+                     Also supported: CMAKE_CXX_FLAGS_DEBUG,
+                     CMAKE_CXX_FLAGS_RELEASE, and CMAKE_CXX_FLAGS_RELWITHDEBINFO
+\end{verbatim}
+\normalsize
 \item Additional QMCPACK build options
+\verbatimfont{\footnotesize}
 \begin{verbatim}
 QMC_INCLUDE         Add extra include paths
 QMC_EXTRA_LIBS      Add extra link libraries
 QMC_BUILD_STATIC    Add -static flags to build
 QMC_DATA            Specify data directory for QMCPACK performance and integration tests
 QE_BIN              Location of Quantum Espresso binaries including pw2qmcpack.x
+ENABLE_MKL          Enable Intel MKL libraries.
```

Comment:
Please do as other libraries. Not mixed with QMC_XXX.
Also mention turn on by default when intel compiler is used.